app/models/users/notification_setting.rb

```diff
@@ -0,0 +1,16 @@
+module Users
+  class NotificationSetting < ApplicationRecord
+    self.table_name_prefix = "users_"
+
```

Comment:
I moved across some validations from the Users models. We cannot delete these validations from the Users tables as yet until we've completely synced the data and have started using the new tables.